controllers/usersController.ts

```diff
@@ -0,0 +1,25 @@
+import { Request, Response } from "express";
+import { Building, BuildingUser, User, Room, sequelize } from "../db/models";
+import { Sequelize } from "sequelize-typescript";
```

Comment:
I think you meant to import the sequelize from models?

Reply:
if you intend to use the transaction, i think is to do sequelize.transaction() with the sequelize referencing from the model instance of sequelize.